### ISSUE TITLE
Problem reproducing the results with your pretrained weights.

### ISSUE THREAD

**Issue by DanielShalam** [lanfenghuanyu/Sill-Net#5]
Hello,
I'm trying to reproduce the results you described using your pre-trained weights on WRN (On few-shot learning tasks).

I extract the .tar file you supply using the save_pkl script, and then I run the PT-MAP code on those features, but the accuracy does not match to one you described.

can you confirm that or supply a different .tar files?

Thank you for your work.

**Comment by lanfenghuanyu** [comment]
Hi,
May you confirm that you used our trained final models, not the pretrained models before our training?
Thank you for your question.

**Comment by DanielShalam** [comment]
Yes I do.
For MiniImagenet I use 'miniImagenet_165.tar' file as the trained final model.

**Comment by lanfenghuanyu** [comment]
We have checked and found no problem.
Are the following parameters in the io_utils.py consistent？
--model=WideResNet28_10
--method=S2M2_R
--num_classes=200
We re-upload the weights as well as the features (.plk), and you may check the features using test_standard.py.
Hope it helps.

**Comment by DanielShalam** [comment]
I can do so using your .plk file.
Thank you!
